setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     docker>=6.0
     websockets>=10.4
     pyhocon
+    omegaconf
```

Comment:
As @yhwen mentioned, if we add them here, pyhocon and omegaconf becomes mandatory.
We should discuss on the pros and cons to make them mandatory.